README.md

```diff
@@ -59,12 +59,10 @@ sudo apt update
 sudo apt install libsdl2-dev
 sudo apt install build-essential
 ```
-To build the engine, editor and tests, use `cmake` and then `make`:
+To build the engine, editor and tests, use CMake from the root directory:
 ```bash
-mkdir build
```

Comment:
Why do you want to complicate the build instructions, and why did you change Release to Debug?